### PR TITLE
fixed typo in MPU->RASR register name

### PR DESCRIPTION
**Pull request by rajszym**

### CODE REVIEW

**Comment by JonatanAntoni** [comment]
Hi @rajszym,

thanks for pointing this out.
May I ask you to
- fix all the typos at once (I'll mark missing ones, individually)
- bump the file version in the header comment

Thanks,
Jonatan

File: CMSIS/Core/Include/mpu_armv7.h [hunk 1]
```diff
@@ -223,7 +223,7 @@ __STATIC_INLINE void ARM_MPU_ClrRegion(uint32_t rnr)
 
 /** Configure an MPU region.
 * \param rbar Value for RBAR register.
-* \param rsar Value for RSAR register.
+* \param rsar Value for RASR register.
```
Comment (JonatanAntoni) [inline CMSIS/Core/Include/mpu_armv7.h:226]: The parameter name has the same typo.

File: CMSIS/Core/Include/mpu_armv7.h [hunk 2]
```diff
@@ -234,7 +234,7 @@ __STATIC_INLINE void ARM_MPU_SetRegion(uint32_t rbar, uint32_t rasr)
 /** Configure the given MPU region.
 * \param rnr Region number to be configured.
 * \param rbar Value for RBAR register.
-* \param rsar Value for RSAR register.
+* \param rsar Value for RASR register.
```
Comment (JonatanAntoni) [inline CMSIS/Core/Include/mpu_armv7.h:237]: The parameter name has the same typo.

**Comment by rajszym** [comment]
typos are only in parameter descriptions

**Comment by JonatanAntoni** [comment]
`* \param *rsar* Value for RASR register.`

**Comment by rajszym** [comment]
done

**Pull request merged**